librz/include/rz_il/definitions/mem.h

```diff
@@ -10,16 +10,31 @@
 extern "C" {
 #endif
 
-struct rzil_mem_t {
-	HtPP *kv_map;
-	ut32 min_unit_size; // minimal unit size in bit (len of value bv)
-};
-typedef struct rzil_mem_t RzILMem;
+typedef unsigned int RzILMemIndex;
```

Comment:
```suggestion
typedef ut32 RzILMemIndex;
```

Reply:
Sounds good, pretty much any type is fine here.